android/src/app/organicmaps/widget/placepage/PlacePageView.java

```diff
@@ -1592,10 +1164,10 @@ private void updateDownloader(CountryItem country)
 
     mDownloaderIcon.update(country);
 
-    StringBuilder sb = new StringBuilder(StringUtils.getFileSizeString(getContext(), country.totalSize));
+    StringBuilder sb = new StringBuilder(StringUtils.getFileSizeString(requireContext(), country.totalSize));
```

Comment:
@arnaudvergnet On the device I have this error:
```
02-02 18:18:25.200 30175 30175 F ganicmaps.debu: runtime.cc:590] Pending exception java.lang.IllegalStateException: Fragment PlacePageView{9d2b38f} (5086c6c7-c27c-4547-a594-a2e8249b8f4f) not attached to a context.
02-02 18:18:25.200 30175 30175 F ganicmaps.debu: runtime.cc:590]   at void app.organicmaps.widget.placepage.PlacePageView.updateDownloader(app.organicmaps.downloader.CountryItem) (PlacePageView.java:1167)
02-02 18:18:25.200 30175 30175 F ganicmaps.debu: runtime.cc:590]   at void app.organicmaps.widget.placepage.PlacePageView.updateDownloader() (PlacePageView.java:1181)
```

And `02-02 18:18:25.200 30175 30175 F ganicmaps.debu: runtime.cc:598] JNI DETECTED ERROR IN APPLICATION: JNI NewObjectV called with pending exception java.lang.IllegalStateException: Fragment PlacePageView{9d2b38f} (5086c6c7-c27c-4547-a594-a2e8249b8f4f) not attached to a context.`

It looks like the downloader code does not work now as before.

Internally it is a complex jni/C++ implementation that downloads map files from C++, but notifies the subscribed Place Page by calling callbacks. Maybe these callbacks are incorrectly handled now.

Reply:
What did you do to get this error?

Reply:
Selected any object on the non-downloaded area, closed PP, selected it again, and pressed Download button in the PP.